### DISCUSSION TITLE
Are the BRNT and CBAR Overlays supported on HDMI=2 output

### DISCUSSION THREAD

**Discussion by jamiefletchertv** [gopro/labs#974] [General]
As the subject states... "Are the BRNT and  Overlays supported on HDMI=2 output"? 

GoPro Hero11 Black
Lab Version 02_30_70

Example QR Code only shows Batter and Card overlays (via HDMI). It is however present in the output MP4. 

So if this is a FR just let me know and ill park it there (as customisation of what overlays you enable would be a great features). 

https://gopro.github.io/labs/control/set/?cmd=mVr1080p60e4!NfW0thS0dR0aScNd1b1wAi16M1S0sMag0v0q0dVoV0B1D0R1C0W0L0oT240703174658.772oTI59291!MWAKE=1!MTUSB=1!MHDMI=1!MBRNO=120!MBURN="(0,20)[BRHH:MM:SSaa mm-dd-yyy]"

**Comment by dnewman-gpsw** [comment]
They are not.

Your command is also problematically long. Don't use settings and active in the same command. Also you change every setting on the camera, the camera remembers it last setting. Are you really changing everything between uses? Also WAKE=2 not WAKE=1, I think you want.
